packages/synapse-react-client/src/components/QueryVisualizationWrapper/QueryVisualizationWrapper.tsx

```diff
@@ -174,7 +174,7 @@ export function QueryVisualizationWrapper(
       const forceDisplayOriginalColumnName =
         localStorage.getItem('force-display-original-column-names') === 'true'
 
-      if (!columnName || forceDisplayOriginalColumnName) {
+      if (!columnName || (forceDisplayOriginalColumnName && !jsonPath)) {
```

Comment:
If we have a `jsonPath` don't use `columnName` even if `forceDisplayOriginalColumnName` is true